### PR TITLE
Changes check for 'bottom of the list', restores fetchyboi, and corrects data

### PR DESCRIPTION
**Pull request by andycarrell**

### CODE REVIEW

File: fetchy/fetchyboi.ts [hunk 1]
```diff
@@ -43,6 +43,9 @@ const extractAndStoreVaccineData = async (rawHtml: string) => {
     vaccineData.dataValidAsAtNzTimeIso
   );
 
+  if (existingData) {
+    console.log(diff(existingData.data, vaccineData));
```
Comment (jishaal) [inline fetchy/fetchyboi.ts:47]: Does this log in github?

**Review by jishaal — APPROVED** [review]

File: data/vaccineData/2021-11-09.json [hunk 2]
```diff
@@ -214,28 +214,8 @@
...
-        "secondDosesPercentage": null,
-        "secondDosesTo90Percent": null,
-        "totalPopulation": null
       }
     ],
     "dataValidAsAtNzTimeIso": "2021-11-09T23:59:00.000+13:00"
   }
-}
\ No newline at end of file
+}
```
Comment (andycarrell) [inline data/vaccineData/2021-11-09.json:221]: damn, format on push - 1 sec

File: fetchy/fetchyboi.ts (hunk 1 above)
Comment (andycarrell) [reply fetchy/fetchyboi.ts:47]: yeh it should do - It was a debug thing though will remove

Comment (jishaal) [reply fetchy/fetchyboi.ts]: Cool cool

**Pull request merged**